app/controllers/concerns/idv_session_concern.rb

```diff
@@ -7,6 +7,8 @@ module IdvSessionConcern
   end
 
   def confirm_idv_needed
+    return if user_needs_selfie?
+
     return if idv_session_user.active_profile.blank? ||
               decorated_sp_session.requested_more_recent_verification? ||
               idv_session_user.reproof_for_irs?(service_provider: current_sp)
```

Comment:
Could we be consistent one way or the other with the mega-"or" vs. one-return-per-line?

```suggestion
    return if user_needs_selfie? ||
              idv_session_user.active_profile.blank? ||
              decorated_sp_session.requested_more_recent_verification? ||
              idv_session_user.reproof_for_irs?(service_provider: current_sp)
```

Or:

```suggestion
    return if user_needs_selfie?
    return if idv_session_user.active_profile.blank?
    return if decorated_sp_session.requested_more_recent_verification? 
    return if idv_session_user.reproof_for_irs?(service_provider: current_sp)
```

Reply:
might be easier if we had one `idv_needed?` or `idv_not_needed?` predicate we could check? (and then have consistent formatting within that?)